spark_ec2.py

```diff
@@ -327,13 +411,19 @@ def parse_args():
     parser.add_option(
         "--instance-profile-name", default=None,
         help="IAM profile name to launch instances under")
+    parser.add_option("-c", "--conf-file", default=None,
+                        help="Specify config file", metavar="FILE")
```

Comment:
Yep, we need to add some more description in the help section. It will be great to document this feature, can we add a pointer to the `README.md` file ?

If `-c/--conf-file` is used, only the arguments present in the configuration file are used and the options present in the command line, will be ignored. It'll nice to add this information somewhere everyone can spot it directly. Can we print a message to the console when `-c/--conf-file` is used, to indicate that only the arguments in the configuration file are used ? Or write it in the `help` section of `add_option` ?

When we will switch from `optparse` to `argparse`, we can easily manage config file and the other command line arguments at the same time.